### PR TITLE
Polkadot Primitives v8 

### PR DESCRIPTION
**Pull request by sandreim**
As Runtime release 1.3.0  includes all of the remaining staging primitives and APIs we can now release primitives version 8.
No other changes other than renaming/moving done here.

### CODE REVIEW

File: polkadot/primitives/src/lib.rs [hunk 1]
```diff
@@ -19,8 +19,8 @@
 #![warn(missing_docs)]
 #![cfg_attr(not(feature = "std"), no_std)]
 
-// `v6` is currently the latest stable version of the runtime API.
-pub mod v7;
+// `v11` is currently the latest stable version of the runtime API.
```
Comment (ordian) [inline polkadot/primitives/src/lib.rs:22]: Not introduced here, but not sure why we need to mentione runtime API version here.

**Review by ordian — APPROVED** [review]

File: polkadot/node/subsystem-util/src/runtime/mod.rs [hunk 2]
```diff
@@ -32,17 +32,21 @@ use polkadot_node_subsystem_types::UnpinHandle;
...
+	Id as ParaId, IndexedVec, NodeFeatures, OccupiedCore, ScrapedOnChainVotes, SessionIndex,
+	SessionInfo, Signed, SigningContext, UncheckedSigned, ValidationCode, ValidationCodeHash,
+	ValidatorId, ValidatorIndex, LEGACY_MIN_BACKING_VOTES,
 };
 
+use std::collections::{BTreeMap, VecDeque};
+
+use crate::{has_required_runtime, request_claim_queue, request_disabled_validators, runtime};
+
 use crate::{
```
Comment (alindima) [inline polkadot/node/subsystem-util/src/runtime/mod.rs:44]: nit: group these together

**Review by alindima — APPROVED** [review]

**Review by tdimitrov — APPROVED** [review]

**Review by alexggh — APPROVED** [review]
Approved, but this technically changes some public import paths in polkadot-primitives and polkadot-subsystem-util, so I think it can't be R0-silent and you need a PrDoc where you mark those crates as a major bump.

**Comment by sandreim** [comment]
> Approved, but this technically changes some public import paths in polkadot-primitives and polkadot-subsystem-util, so I think it can't be R0-silent and you need a PrDoc where you mark those crates as a major bump.

Shouldn't break anyone at code level, this stuff was already released.

**Pull request merged**